hooks/useFileUpload.tsx

```diff
@@ -18,11 +21,10 @@ const useFileUpload = () => {
 
     try {
       const file = event.target.files[0]
+      if (!file) throw new Error()
+      const JWT = await getIpfsJwt(address)
```

Comment:
what would be the side effects of removing the `address` param from this function call?